plugins/newsdownloader.koplugin/main.lua

```diff
@@ -97,12 +113,49 @@ function NewsDownloader:addToMainMenu(menu_items)
             },
             {
                 text = _("Settings"),
-                callback = function()
-                    UIManager:show(InfoMessage:new{
-                        text = T(_("To change feed (Atom/RSS) sources please manually edit the configuration file:\n%1\n\nIt is very simple and contains comments as well as sample configuration."),
-                                 feed_config_path)
-                    })
-                end,
+                sub_item_table = {
+                    {
+                        text = _("Change feeds configuration"),
+                        callback = function()
+                            UIManager:show(InfoMessage:new{
+                                text = T(_("To change feed (Atom/RSS) sources please manually edit the configuration file:\n%1\n\nIt is very simple and contains comments as well as sample configuration."),
+                                         feed_config_path)
+                            })
+                        end,
+                    },
+                    {
+                        text = _("Action when Wi-Fi is off"),
+                        sub_item_table = {
+                            {
+                                text = _("Turn on"),
+                                checked_func = function()
+                                    return nd_action_when_wifi_off() == "turn_on"
+                                end,
+                                callback = function()
+                                    G_reader_settings:saveSetting("nd_action_when_wifi_off", "turn_on")
```

Comment:
This settings name is extremely complex.

Reply:
Sorry, I should've been clearer. I was bothered more by the "hard to understand" than the "complex" per se. That's even less comprehensible for someone trying to read the code.

Keep wifi, get rid of nd — what is nd? (named?) and please call it something clear and simple like `wifi_enable_action`.

Reply:
Ok. Could your be also clearer, comment above?